File: LeetCode_Algorithm/1822(Sign of the Product of an Array)/1822.py
class Solution(object):
    def arraySign(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        # sol1 45ms 54% 13.3MB 90%
        ans = 1
        for num in nums:
            if num == 0:
                return 0
            elif num < 0:
                ans *= -1
        return ans

File: LeetCode_Algorithm/1822(Sign of the Product of an Array)/1822.js
var arraySign = function(nums) {
  // sol1 49ms 96% 44.2MB 30%
  let ans = 1
  for (let num of nums) {
      if (num == 0) {
          return 0
      } else if (num < 0) {
          ans *= -1
      }
  }
  return ans
};